src/Pact/Types/ChainMeta.hs

```diff
@@ -75,16 +77,30 @@ instance FromJSON PrivateMeta where parseJSON = lensyParseJSON 3
 instance NFData PrivateMeta
 instance Serialize PrivateMeta
 
+
+-- | Contains ChainId, Chainweb network version, ect.
+--   Meant to prevent sending Testnet txs to Mainet.
+newtype ChainId = ChainId Text
+  deriving (Eq, Show, Generic)
+
+instance ToJSON ChainId
+instance FromJSON ChainId
+instance NFData ChainId
+instance Serialize ChainId
+instance Default ChainId where def = ChainId "Testnet00/0"
```

Comment:
Don't supply a `Default` instance; do derive an `IsString` instance so we can issue literals.

Reply:
Done